app/AppKernel.php

```diff
@@ -32,6 +32,12 @@
 
 class AppKernel extends Kernel
 {
+    const VERSION = '1.7.4.0';
```

Comment:
I already wanted to introduce this but @eternoendless was against this change because this makes the release harder to do (not that hard, but still^^).

So before continue, we need @eternoendless approval.